bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -23,7 +23,8 @@
     CONFIGURATION_ERROR(7),
     BRIDGE_OFFLINE(8),
     FIRMWARE_UPDATING(9),
-    DUTY_CYCLE(10);
+    FIRMWARE_UPDATE_PENDING(10),
```

Comment:
What exactly is the semantics of this new status? Does it mean that a new firmware is available? Does it mean that the handler is instructed to transfer it?
Note that you also need to update https://github.com/eclipse/smarthome/blob/master/docs/documentation/concepts/things.md#status-details.